packages/authentication/test/unit/provider.test.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {Context, Provider, ValueOrPromise, instantiateClass} from '@loopback/context';
```

Comment:
AFAICT, `ValueOrPromise` is not used in this test. Can you please remove it from this list?